db/migrate/20161026004331_remove_duplicate_versions.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class RemoveDuplicateVersions < ActiveRecord::Migration[5.0]
+  def up
+    # delete all versions that are not unique
+    PaperTrail::Version.connection.execute <<-SQL
+      delete from versions where id not in (
+        SELECT * FROM (select min(id) from versions group by item_id, item_type, object) AS temp_tab
+      )
```

Comment:
Have you tried running a variation of this query to find out how many rows would be deleted?